syntax = "proto3";

package AcFunDanmu.Im.Cloud.Message;

message Image {
	string uri = 1;
	int32 width = 2;
	int32 height = 3;
	int64 contentLength = 4;
}
